.gitignore

```diff
@@ -57,3 +57,6 @@ target/
 .idea/
 
 test.sh
+
+# ignoring virtualenvironment
```

Comment:
This does not belong here, it's only relevant to your specific choice. Please remove.
As a side note, PEP8 requires an empty line at the end of the file.